app/src/androidTest/java/com/github/se/wanderpals/dashboard/DashboardTest.kt

```diff
@@ -353,10 +353,10 @@ class DashboardTest : TestCase(kaspressoBuilder = Kaspresso.Builder.withComposeS
       Dashboard(tripId = "", dashboardViewModel = viewModel, navActions = mockNavActions)
     }
     composeTestRule.onNodeWithTag("menuButton").performClick()
-    composeTestRule.onNodeWithTag("MemberListMenu").performClick()
+    composeTestRule.onNodeWithTag("AdminButtonTest").performClick()
     composeTestRule.onNodeWithTag("menuNav").assertIsNotDisplayed()
 
-    verify { mockNavActions.navigateTo(Route.MEMBERS) }
```

Comment:
I think you should delete the Route.MEMBERS entirely since you have merged his screen with yours

Reply:
Yes, I forgot to remove it from the Route enum class